addons/trivy-operator/README.md

```diff
@@ -0,0 +1,74 @@
+# trivy-operator
+
+## Install
+
+```shell
+vela addon enable trivy-operator
+```
+
+## Uninstall
+
+```shell
+vela addon disable trivy-operator
+```
+
+## Quick start
+```shell
+ vela ls -A | grep trivy
+vela-system     addon-trivy-operator    trivy-operator          helm                    running healthy Fetch repository successfully, Create helm release                  2022-06-13 21:15:17 +0800 CST
+
+vela status addon-trivy-operator -n vela-system
+About:
+
+  Name:         addon-trivy-operator         
+  Namespace:    vela-system                  
+  Created at:   2022-06-13 21:15:17 +0800 CST
+  Status:       running                      
+
+Workflow:
+
+  mode: StepByStep
+  finished: true
+  Suspend: false
+  Terminated: false
+  Steps
+  - id:58dtg4z65x
+    name:deploy-control-plane
+    type:apply-application
+    phase:succeeded 
+    message:
+  - id:j9p8mem2m9
+    name:deploy-runtime
+    type:deploy2runtime
+    phase:succeeded 
+    message:
+
+Services:
+
+  - Name: trivy-operator  
+    Cluster: local  Namespace: vela-system
+    Type: helm
+    Healthy Fetch repository successfully, Create helm release successfully
+    No trait applied
+
+vela port-forward addon-trivy-operator -n vela-system
+Forwarding from 127.0.0.1:9115 -> 9115
+Forwarding from [::1]:9115 -> 9115
+Forward successfully! Opening browser ...
+
+kubectl label namespaces vela-system trivy-scan=true
+kubectl label namespaces vela-system trivy-operator-validation=true
```

Comment:
this command should be something like  :

```
vela env set vela-system --label trivy-scan=true
```

Then we can tell users that this is a environment for development which can scan vulnerabilities.